apps/guide/src/content/04-slash-commands/05-autocomplete.mdx

```diff
@@ -0,0 +1,250 @@
+---
+title: Autocomplete
+category: Slash commands
+---
+
+# Autocomplete
+
+Autocomplete allows you to dynamically provide a selection of values to the user, based on their input, rather than relying on static choices. In this section we will cover how to add autocomplete support to your commands.
+
+<Alert title="Read first!" type="info">
+	This page is a follow-up to the [slash commands](./advanced-command-creation) section covering options and option
+	choices, and assumes you have also implemented [event handling](../creating-your-bot/event-handling). Please carefully
+	read those pages first so that you can understand the methods used in this section.
+</Alert>
+
+## Enabling autocomplete
+
+To use autocomplete with your commands, _instead_ of listing static choices, the option must be set to use autocompletion. This is done by setting `_autocomplete: true_` in the <DocsLink type="typedef" parent="ApplicationCommandOptionData" />. Autocomplete can be used on the same option types that support static choices; _`String`_, _`Integer`_ and _`Number`_.
```

Comment:
```suggestion
To use autocomplete with your commands, _instead_ of listing static choices, the option must be set to use autocompletion. This is done by setting _`autocomplete: true`_ in the <DocsLink type="typedef" parent="ApplicationCommandOptionData" />. Autocomplete can be used on the same option types that support static choices; _`String`_, _`Integer`_, and _`Number`_.
```
assuming this needs to be inverted - have not really worked with the new format yet
![image](https://github.com/discordjs/discord.js/assets/26532370/03c1229f-6083-485b-abb9-63f35f77e182)